src/Client.cpp

```diff
@@ -26,7 +26,7 @@
 
 namespace voltdb {
 
-Client Client::create(ClientConfig config) throw (voltdb::Exception, voltdb::LibEventException) {
+Client Client::create(ClientConfig config) {
```

Comment:
Why didn't you use this factory method but exposed the `ClientImpl`? What was the benefit?